examples/src/main/java/com/rvandoosselaer/blocks/examples/BlockBuilder.java

```diff
@@ -186,11 +189,13 @@ private void updatePlaceholders(CollisionResult result) {
     }
 
     private void addBlock() {
-        chunkManager.addBlock(addPlaceholder.getWorldTranslation().subtract(0.5f, 0.5f, 0.5f), blockRegistry.get(BlockIds.GRASS));
+        Vector3f blockLocation = addPlaceholder.getWorldTranslation().mult(1f / BlocksConfig.getInstance().getBlockScale());
+        chunkManager.addBlock(blockLocation.subtract(0.5f, 0.5f, 0.5f), blockRegistry.get(BlockIds.GRASS));
     }
 
     private void removeBlock() {
-        chunkManager.removeBlock(removePlaceholder.getWorldTranslation().subtract(0.5f, 0.5f, 0.5f));
+        Vector3f blockLocation = removePlaceholder.getWorldTranslation().mult(1f / BlocksConfig.getInstance().getBlockScale());
+        chunkManager.removeBlock(blockLocation.subtract(0.5f, 0.5f, 0.5f));
```

Comment:
Note, these changes in `addBlock()`  and `removeBlock()` are not needed, no need to divide WorldTranslation by block scale.

Reply:
You are correct. I first didn't adapt the addBlock and removeBlock implementations and did it in the example itself. After changing the implementations I forgot to remove this in the example.

Thanks! 🎉